thousandeyes-sdk-core/src/thousandeyes_sdk/core/api_client.py

```diff
@@ -190,7 +182,7 @@ def param_serialize(
                 # specified safe chars, encode everything
                 resource_path = resource_path.replace(
                     '{%s}' % k,
-                    quote(str(v), safe=config.safe_chars_for_path_param)
+                    quote(str(v), safe='')
```

Comment:
do we still need this replace logic?

Reply:
this is making sure every thing is encoded in the URL which I don't think we ever need, so I'd say this can also be removed

Reply:
Ah no, I think we should maintain it, wasn't understanding the logic behind it